React-Projects/job-portal/src/EditJob.jsx

```diff
@@ -3,34 +3,46 @@ import { useNavigate } from "react-router-dom";
 import "./styles/EditJob.scss";
 import { editJob } from "./api";
 import { JWT_COOKIE } from "./Constants";
+import { toast } from "react-toastify";
 
 const EditJob = () => {
-  let isLoggedIn = false;
+  const [jobDetails, setJobDetails] = useState({
+    jobTitle: "",
+    jobType: "SOFTWARE ENGINEERING",
+    jobSalary: 0,
+    jobDescription: "",
+  });
+
+  const handleChange = (e) => {
+    e.preventDefault();
+    setJobDetails({
+      ...jobDetails,
+      [e.currentTarget.name]: e.currentTarget.value,
+    });
+  };
+
   const navigate = useNavigate();
   const urlParams = new URLSearchParams(window.location.search);
 
   const jobId = urlParams.get("id");
 
-  if (JWT_COOKIE) {
-    isLoggedIn = true;
-  }
-
-  const [jobTitle, setJobTitle] = useState("");
-  const [jobType, setJobType] = useState("SOFTWARE ENGINEERING");
-  const [jobSalary, setJobSalary] = useState(0);
-  const [jobDescription, setJobDescription] = useState("");
-
   const editAJob = async (event) => {
     event.preventDefault();
     try {
-      await editJob(jobId, jobTitle, jobType, jobSalary, jobDescription);
-      alert("Edited the job");
+      await editJob(
```

Comment:
@codal-kcolaco here you can use then and catch block instead of try catch and await-sycn.